code/modules/mining/abandonedcrates.dm

```diff
@@ -198,15 +197,15 @@
 		if(guesschar != code[i])
 			. = 0
 
-/obj/structure/closet/crate/secure/loot/attackby(obj/item/W as obj, mob/user as mob)
+/obj/structure/closet/crate/secure/loot/attackby(obj/item/W, mob/user)
 	if(locked)
-		if (W.ismultitool()) // Greetings Urist McProfessor, how about a nice game of cows and bulls?
-			to_chat(user, "<span class='notice'>DECA-CODE LOCK ANALYSIS:</span>")
-			if (attempts == 1)
-				to_chat(user, "<span class='warning'>* Anti-Tamper system will activate on the next failed access attempt.</span>")
+		if(W.ismultitool()) // Greetings Urist McProfessor, how about a nice game of cows and bulls?
+			to_chat(user, SPAN_NOTICE("DECA-CODE LOCK ANALYSIS:"))
+			if(attempts)
```

Comment:
Bad change.
If you change that, you will always get the message that the anti-tamper system will activate on the next failed attempt.